### DISCUSSION TITLE
7Z and Preferences Import/Export

### DISCUSSION THREAD

**Discussion by mmtka** [99natmar99/Windows-11-Fixer#5] [Ideas]
Hey mate,

your fixer is out of the space and I love that!!! 

Would be possible to add in Install tab in "Another" 7z as well?

And I have an idea, if it would be possible to save a preferences file wiht your own preferences that you could export/import when you want to do the same actions on mor PCs.

Anyway thank you for your brilliant work that saves loooooot of time. 🥇

**Comment by 99natmar99** [comment]
7-Zip will be included in the next update.

A preferences file is a great idea and I will consider it for a future iteration of the program.

I'm glad to hear you enjoy the program!

**Comment by mmtka** [comment]
And I'm glad to hear / read so positive answer. Thnx
